src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String name;
+    protected boolean isDone;
+
+    public Task(String name) {
+        this.name = name;
+        isDone = false;
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    protected String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Should the comment `//return tick or X symbols` be moved to a new line before return (isDone ? "\u2713" : "\u2718"); and changed to `// Return ...` to follow [this convention](https://se-education.org/guides/conventions/java/basic.html#comments)?